airgun/session.py

```diff
@@ -154,6 +154,7 @@ def __init__(
         session_cookie=None,
         url=None,
         login=True,
+        hostname=None,
```

Comment:
I think that should be before `url`, since URI is constructed by appending `url` to `hostname`.
Remember to also change docstring.

Reply:
I decided on this order so anyone using the session object with positional arguments wouldn't be affected. If we can determine that there are no instances of positional-only arguments, then I can move it up.
Docstring has been updated

Reply:
Fair concern. Fortunately, there are no positional-only arguments past `password`. `session_cookie` effectively forces everything later to be keyword-only, as nothing in Robottelo actually uses `session_cookie` (but people may use it locally to save some time during test development).

Checked with following commands in Robottelo root dir:
```bash
$ ack 'session\((.*?, ?){3}'
$ ack 'session\('
```

iqe-satellite-plugin only ever passes test name.

Reply:
if you say it is safe, then I can amend my latest changes